events-api/src/modules/tickets/dto/requests/tickets-create-request-body.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from "zod";
+
+export const ticketsCreateRequestBodyDtoSchema = z.object({
+  ticketQuantity: z
+    .number({ message: "Ticket quantity must be number" })
+    .int({ message: "Ticket quantity must be integer" })
+    .positive({ message: "Ticket quantity must be positive" }),
+  ticketPrice: z
+    .number({ message: "Ticket price must be number" })
+    .positive({ message: "Ticket price must be positive" }),
+  eventId: z.string().uuid({ message: "Event ID is not a valid uuid" }),
+});
+
+export type ticketsCreateRequestBodyDto = z.infer<
```

Comment:
Fix casing for type name, prefer PascalCase

Reply:
fixed